data_preparation.R

```diff
@@ -310,10 +311,23 @@ data_trend <- comb_all(data_trend_op,
   # This will be used as the x-axis in a shiny plot
   mutate(quarter_date_last = as.yearmon(quarter_date,
                                         "%d/%m/%Y")) %>%
+  # removing National Facility All Outpatient Appointments
+  filter(loc_code != "SB0801") %>%
+  # distinguishing between GJNH and NHS Louisa Jordan
+  mutate(hb_name = case_when(hb_name == "National Facility" ~ loc_name,
+                             TRUE ~ hb_name)) %>%
   
   # Arrange by date for later plotting
   arrange(quarter_date_last)
 
+### recoding GJNH and Louisa Jordan to NHS Boards
+nf_data_trend = data_trend %>% 
+  filter(hb_code == "SB0801") %>%
```

Comment:
Does the filter bring anything back because NF has already been removed from the data_trend tibble?

Reply:
Only activity where National Facility was listed as both loc_code and hb_code was removed previously - GJNH and LJ will still be under NF health board.

Reply:
Was the health board name not changed to the loc_name as above?

Reply:
Yes, the hb names were changed from NF to GJNH or LJ but the code stayed the same, which is what I filter on.